nvflare/private/fed/client/client_engine.py

```diff
@@ -139,15 +141,13 @@ def start_app(
         if not os.path.exists(app_root):
             return f"{ERROR_MSG_PREFIX}: Client app does not exist. Please deploy it before starting client."
 
-        if self.client.enable_byoc:
-            app_custom_folder = os.path.join(app_root, "custom")
+        app_custom_folder = os.path.join(app_root, "custom")
+        if os.path.isdir(app_custom_folder):
```

Comment:
should we use something like `app_custom_folder = workspace.get_app_custom_dir(run_number)`

Reply:
Will address this in another PR